api/src/routing.js

```diff
@@ -12,15 +12,28 @@ const prometheusMiddleware = require('prometheus-api-metrics');
 const rateLimiterMiddleware = require('./middleware/rate-limiter');
 const logger = require('./logger');
 const isClientHuman = require('./is-client-human');
-const target = 'http://' + environment.host + ':' + environment.port;
-const proxy = require('http-proxy').createProxyServer({ target: target });
+
+const isString = value => typeof value === 'string' || value instanceof String;
+const isTrue = value => isString(value) ? value.toLowerCase() === 'true' : value === true;
```

Comment:
Move these to environment.js and then call them when reading the process.env so we just store the boolean in memory rather than executing this function every time we want to check.

Reply:
Will do.

Reply:
I'll look at it.

Reply:
By these do you mean the ultimate value for the process.env? Not sure I follow if you're referencing the functions as they're initialised in the file scope and initialised once at `server.js`. Happy to move the variables to  `environment.js` though.

Reply:
Ahh yes I see what you mean. Now that it's in environment I think this is clearer.